Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "actix-web-validator"
 version = "1.0.1-alpha.0"
```

Comment:
Should we change the version?

Reply:
After cargo-release it upgrades automagically.